libgalois/include/katana/PropertyGraph.h

```diff
@@ -29,12 +29,81 @@ struct KATANA_EXPORT GraphTopology {
   using edges_range = StandardRange<edge_iterator>;
   using iterator = node_iterator;
 
+private:
```

Comment:
nit: public member functions first, then public variables, then private member functions, and finally private variables.

Reply:
done